src/kOS/Suffixed/VesselTarget.cs

```diff
@@ -464,6 +465,13 @@ private void InitializeSuffixes()
             AddSuffix("MESSAGES", new NoArgsSuffix<MessageQueueStructure>(() => GetMessages()));
         }
 
+        public global::Part GetControlPart()
+        {
+            global::Part res = Vessel.GetReferenceTransformPart(); //this can actually be null if "control from here" has never been used
```

Comment:
Must be some wonders with legacy code stemming from the times when you could have only one command module on the ship and it was always the root.